Mlem/App/Views/Shared/Search/SearchView+Views.swift

```diff
@@ -227,4 +261,22 @@ extension SearchView {
             }
         }
     }
+    
+    @ViewBuilder
+    func deleteRecentSearchButton(session: UserSession, callback: @escaping (() -> Void)) -> some View {
+        if editingRecentSearches {
+            Button {
+                withAnimation {
+                    callback()
+                }
+                Task(priority: .background) {
+                    try await session.saveVisitHistory()
+                }
+            } label: {
+                Image(systemName: Icons.delete)
```

Comment:
The accessibility-compatible way to do this would be

```suggestion
                Label("Remove Recent Search", systemImage: Icons.delete)
                    .labelStyle(.iconOnly)
```

Or alternatively, if the padding is able to go outside of the button, something like this

```swift
Button("Remove Recent Search", systemImage: Icons.delete) {
    withAnimation {
        callback()
    }
    Task(priority: .background) {
        try await session.saveVisitHistory()
    }
}
.labelStyle(.iconOnly)
 .foregroundStyle(palette.negative)
```

Mlem as a whole probably isn't very VO-compatible at the moment but it doesn't hurt to get into the habit of making things accessible